react/PlaceDetails.tsx

```diff
@@ -50,17 +63,24 @@ const PlaceDetails: React.FC<Props> = ({
               : address[fragment.name]
 
             return (
-              <span key={fragment.name}>
+              <div className="dib" key={fragment.name}>
                 {fragment.delimiter && hasPreviousFragment && (
                   <span>{fragment.delimiter}</span>
                 )}
                 <span className={fragment.name}>{addressValue}</span>
                 {fragment.delimiterAfter && shouldShowDelimiter && (
                   <span>{fragment.delimiterAfter}</span>
                 )}
-              </span>
+              </div>
             )
           })}
+          {onEdit && displayIndex === 0 && (
```

Comment:
If `!address[fragment.name] || hiddenFields.includes(fragment.name)` is true, the edit button will appear alone with it's left margin:

print
|-
|<img width="300" src="https://user-images.githubusercontent.com/26108090/98979644-59167500-24fa-11eb-8ecb-9b44f765b4c7.png" />

Can you verify how this case should be handled?

Reply:
sure

Reply:
fixed on 2e47258

Reply:
Nice!! I'll review soon (or ping me when you finish/stop working on the billing workspace)